addons/property_panel/float_slider/float_slider.gd

```diff
@@ -122,7 +125,14 @@ func _on_text_changed(new_text : String) -> void:
 
 
 func _on_text_entered(new_text : String) -> void:
-	value = new_text.to_float()
+	if allow_expressions:
+		var error = _expression.parse(new_text)
+		if error != OK:
+			push_error(_expression.get_error_text())
```

Comment:
I would reserve `push_error` for actual game logic errors, not to log user errors. I think Godot handles this by reverting to the last correct value.